### PR TITLE
changes for pdf2

### PR DESCRIPTION
**Pull request by Flegmatik89**
tasks 1 - 7

### CODE REVIEW

File: src/pdf2/task8/Main.java [hunk 1]
```diff
@@ -15,6 +15,40 @@
 
 public class Main {
     public static void main(String[] args) {
+        int maxSymbols = 4;
 
+        for (int i = 0; i <= maxSymbols; i++) {
+
```
Comment (vovakor1) [inline src/pdf2/task8/Main.java]: extra line

File: src/pdf2/task8/Main.java [hunk 2]
```diff
@@ -15,6 +15,40 @@
...
+            for (int k = maxSymbols; k > i; k--) {
+                System.out.print(" ");
+            }
+
+            for (int j = 0; j < i; j++) {
+                System.out.print("*");
+            }
+
+//            for (int k = maxSymbols; k >= i; k--) {
```
Comment (vovakor1) [inline src/pdf2/task8/Main.java]: Extra comment

File: src/pdf2/task8/Main.java [hunk 3]
```diff
@@ -15,6 +15,40 @@
...
+
+            for (int k = maxSymbols; k > i; k--) {
+                System.out.print(" ");
+            }
+
+            for (int j = 0; j < i; j++) {
+                System.out.print("*");
+            }
+
+//            for (int k = maxSymbols; k > i; k--) {
```
Comment (vovakor1) [inline src/pdf2/task8/Main.java]: extra comments

File: src/pdf2/task8/Main.java [hunk 4]
```diff
@@ -15,6 +15,40 @@
...
+//                System.out.print(" ");
+//            }
+
+            System.out.println(" ");
+
```
Comment (vovakor1) [inline src/pdf2/task8/Main.java]: extra line

File: src/pdf2/task8/Main.java [hunk 5]
```diff
@@ -15,6 +15,40 @@
...
+            }
+
+//            for (int k = maxSymbols; k >= i; k--) {
+//                System.out.print(" ");
+//            }
+
+            System.out.println(" ");
+        }
+        for (int i = maxSymbols; i >= 0; i-- ) {
+
```
Comment (vovakor1) [inline src/pdf2/task8/Main.java]: extra line

File: src/pdf2/task8/Main.java [hunk 6]
```diff
@@ -15,6 +15,40 @@
```
Comment (vovakor1) [inline src/pdf2/task8/Main.java:15]: Make ctrl+alt+L

**Review by vovakor1 — APPROVED** [review]
Resolve comments and merge

File: src/pdf2/task8/Main.java (hunk 6 above)
Comment (Flegmatik89) [reply src/pdf2/task8/Main.java:15]: done

File: src/pdf2/task8/Main.java (hunk 5 above)
Comment (Flegmatik89) [reply src/pdf2/task8/Main.java]: done

File: src/pdf2/task8/Main.java (hunk 4 above)
Comment (Flegmatik89) [reply src/pdf2/task8/Main.java]: done

File: src/pdf2/task8/Main.java (hunk 3 above)
Comment (Flegmatik89) [reply src/pdf2/task8/Main.java]: done

File: src/pdf2/task8/Main.java (hunk 2 above)
Comment (Flegmatik89) [reply src/pdf2/task8/Main.java]: done

File: src/pdf2/task8/Main.java (hunk 1 above)
Comment (Flegmatik89) [reply src/pdf2/task8/Main.java]: done

**Pull request merged**